src/entities/mellow/config/config.ts

```diff
@@ -39,6 +40,7 @@ const networkConfigurations: { [key: string]: NetworkConfiguration } = {
           show: true,
           soon: false,
           deprecated: false,
+          withdrawable: false,
```

Comment:
this should be per vault, not per router

Reply:
yes, resolved.